source/standards/programming-languages.html.md.erb

```diff
@@ -98,6 +98,24 @@ We used Scala in the early days of GDS. GOV.UK Licensing is the only remaining
 application written in Scala but we’ve found it hard to support because of a lack
 of skills in GDS. Do not use Scala for new projects.
 
+## Mobile Development
+
+For developing mobile apps, we use Swift for iOS and Kotlin for Android.
```

Comment:
@mcgarj in line with other docs, consider annotating some of the brand names here with links back to the product pages of that stuff.

So Swift, Kotlin, hell even iOS and Android.

Imagine you're reading this as a dev who (ahem... perhaps like me a few months ago) has zero starting point for this. Let's link richly through to the right pages so someone can go find out what we mean by Swift, Android or even iOS.